roles/static_inventory/defaults/main.yml

```diff
@@ -19,3 +19,6 @@ private_ssh_key: ~/.ssh/openshift
 
 # The patch to store the generated config to access bastion/hosts
 ssh_config_path: /tmp/ssh.config.ansible
+
+openstack_region: regionOne
```

Comment:
The os_client_config module sets it up. If you run the playbook below, it will print out the current region name.

And yes, please set the default to that value (note that it can be an empty string, too and that's okay). And actually, I don't think this makes any sense to be configurable in our playbooks so you can remove the `openstack_region` var.

```
---
- hosts: localhost
  become: false
  gather_facts: false
  tasks:
  - os_client_config:
  - debug: var=openstack.clouds[0].region_name
```